pkg/splunk/enterprise/standalone_test.go

```diff
@@ -1281,7 +1281,11 @@ func TestStandaloneWithReadyState(t *testing.T) {
 		Name:      standalone.Name,
 		Namespace: standalone.Namespace,
 	}
-
+	err = c.Get(ctx, namespacedName, &standalone)
+	if err != nil {
+		t.Errorf("Unexpected get standalone %v", err)
+		debug.PrintStack()
```

Comment:
here too...

Reply:
possible if API server do not respond. this is in rest case. if it happens I wanted add debug message make sure to get stack frame.